### ISSUE TITLE
APEChain 

### ISSUE THREAD

**Issue by sportypawsnft** [nftstorage/nft.storage#2769]
Hi 
Could you add APEChain to the list of NFT Chains ?

**Comment by elizabeth-griffiths** [comment]
YES! Thank you so much for asking. Will prioritise this and let you know
here once it's done.

**Comment by elizabeth-griffiths** [comment]
Hi - We will go live with support for ApeChain this week. Keep an eye on our twitter and please help us get the word out.
